docs/demo/materials/03-atoms/accordions/01-accordion.html

```diff
@@ -0,0 +1,43 @@
+---
+name: Accordion
+title: accordion
+notes: |
+    Use accordions to separate content into logical sections.
+
+    `dc-accordion__item` is logically separated into two sections `dc-accordion__header` and `dc-accordion__content`
+---
+
+<div class="dc-accordion">
+	<div class="dc-accordion__item">
+		<label class="dc-accordion__header">
```

Comment:
As we do not use inputs anymore, can we change this `label` element to `header` to keep it more symantic ?